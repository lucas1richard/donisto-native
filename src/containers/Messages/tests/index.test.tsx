import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Messages, IMessagesProps } from '../index';

describe('<Messages />', () => {
  const renderComponent = (props: IMessagesProps) => toJson(shallow(<Messages {...props} />));
  it('Expect to have unit tests specified', () => {
    expect(renderComponent({})).toMatchSnapshot();
  });
});
