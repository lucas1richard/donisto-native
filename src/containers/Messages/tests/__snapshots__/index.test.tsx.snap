// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Messages /> Expect to have unit tests specified 1`] = `
<Screen>
  <Component
    style={
      Object {
        "alignItems": "center",
        "borderBottomColor": "#d3d3d3",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "height": 60,
        "justifyContent": "space-between",
        "paddingHorizontal": 20,
      }
    }
  >
    <Icon
      allowFontScaling={false}
      color="rgb(251, 100, 65)"
      name="plus"
      size={25}
    />
    <Txt
      color="default"
      style={
        Object {
          "fontSize": 22,
          "fontWeight": "600",
        }
      }
    >
      Messages
    </Txt>
    <Icon
      allowFontScaling={false}
      color="rgb(251, 100, 65)"
      name="magnifying-glass"
      size={25}
    />
  </Component>
  <ScrollViewMock>
    <Component
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <MessageList />
    </Component>
  </ScrollViewMock>
</Screen>
`;
